rllib/algorithms/tests/test_registry.py

```diff
@@ -18,6 +20,15 @@ def test_get_eager_traced_class_name(self):
         traced = PPOTF2Policy.with_tracing()
         self.assertEqual(get_policy_class_name(traced), "PPOTF2Policy")
 
+    def test_registered_algorithm_names(self):
+        """All RLlib registered algorithms should have their name listed in the
+        registry dictionary."""
+
+        for class_name in ALGORITHMS_CLASS_TO_NAME.keys():
+            registered_name = ALGORITHMS_CLASS_TO_NAME[class_name]
+            algo_class, _ = ALGORITHMS[registered_name]()
+            self.assertEqual(class_name, algo_class.__name__)
```

Comment:
So was the bug here that some of the algorithm names did not previously match the class names when they were registered?